introduction/related-work.tex

```diff
@@ -38,7 +38,8 @@ \subsection{Programming Language}
 
 Lower-level languages have usually been the preferred ones for programming of embedded systems.
 Traditionally, a large portion of code bases consists of {\C}, \lang{Assembly}, and {\Cpp}.
-As seen in \autoref{fig:vdc:langs} according to VDC Research \cite{web:vdc} survey language used in embedded systems, the usage of C and assembly are on the way down in favor for higher level languages like C++, Java, and C\#.
+\autoref{fig:vdc:langs} shows a VDC Research \cite{web:vdc} survey over language used in embedded systems.
+According to the survey the usage of C and assembly are on the way down in favor of higher level languages like C++, Java, and C\#.
```

Comment:
`According to the survey, the usage of {\C} and assembly are on the way down in favor of higher level languages like \lang{C++}, {\Java}, and \lang{C\#}.`
Legg til komma og fiks lang :rabbit: